src/current/v23.1/import-performance-best-practices.md

```diff
@@ -7,61 +7,24 @@ docs_area: migrate
 
 This page provides best practices for optimizing [import](import-into.html) performance in CockroachDB.
```

Comment:
I think a sentence about when to use import is warranted. "IMPORT is the fastest method to ingest data into CockroachDB but it requires taking the target table offline for the duration of the IMPORT. IMPORT is a good choice for initial data migrations and data migrations that can tolerate table downtime. If you cannot tolerate table unavailability, we recommend using COPY instead."